taichi/runtime/llvm/llvm_runtime_executor.cpp

```diff
@@ -112,6 +112,15 @@ LlvmRuntimeExecutor::LlvmRuntimeExecutor(CompileConfig &config,
   }
 #endif
 
+#if defined(TI_WITH_AMDGPU)
+  if (config.arch == Arch::amdgpu) {
+    AMDGPUContext::get_instance().set_debug(config.debug);
```

Comment:
1. This does affect performance, but in debug mode, I think this is necessary, otherwise it will make it more difficult to locate the problem
2. By the way, I think the main reason of poor performance on AMDGPU backend is similar to this. And I will try to solve this problem when i finish to pull these code.